bootstrap/aws-public/config-default.sh

```diff
@@ -13,6 +13,8 @@ export TF_VAR_key_name=${TF_VAR_key_name:-apollo}
 # so no really needed to export them. Exporting for consitency.
 export TF_VAR_atlas_artifact_master=${TF_VAR_atlas_artifact_master:-capgemini/apollo-ubuntu-14.04-amd64}
 export TF_VAR_atlas_artifact_slave=${TF_VAR_atlas_artifact_slave:-capgemini/apollo-ubuntu-14.04-amd64}
+export TF_VAR_atlas_artifact_version_master=${TF_VAR_atlas_artifact_version_master:-1}
```

Comment:
As we'll be using a generic terraform apply for all the clouds, yes it is necessary. Also it is working bacuse we are relying on variables.tf in some cases and conofig-default.sh for others, we should probably have a better convention/consitency. Also we could refactor config-default files in a follow up pr as we have a lot of duplication.